### DISCUSSION TITLE
v6.1.0 release

### DISCUSSION THREAD

**Discussion by grepcats** [fabric8io/kubernetes-client#4281] [Q&A] [answered]
Hi all! I know v6.0.0 just dropped, but I'm wondering if there are any thoughts as to the release timeline for v6.1.0? In particular looking for https://github.com/fabric8io/kubernetes-client/pull/4264.

Thanks all! Great work 🎉

**Comment by manusa** [comment, marked as answer]
Hi @grepcats,

It shouldn't take long (weeks, not months).
Now that we have finally released v6.0.0 we should get back to the short release cycles we followed with v5.x.

Regarding #4264, it's a good start, but it doesn't cover many scenarios. We'll merge ASAP, but there's still work to do in that area.

**Reply by grepcats** [reply]
Thank you for the answer & the additional context, @manusa!
